infra/test_build_change.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import sys
+
+import build_and_push_test_images
+
+
+def main():
```

Comment:
@jonathanmetzman Was this intended to be committed? If so, would you mind fixing the lint issues in this file? 
I'm getting lint errors on this from another PR.

Reply:
Didn't mean to commit this. Removed in #6380.